pandas/tests/groupby/test_groupby.py

```diff
@@ -2031,12 +2031,12 @@ def test_mutate_groups(self):
 
         def f_copy(x):
             x = x.copy()
-            x['rank'] = x.val.rank(method='min')
```

Comment:
These warnings will get fixed by #17298

Reply:
Good, I will remove 5425137 from the pull request, it wasn't really complete